.github/PULL_REQUEST_TEMPLATE/bug_fix_template.md

```diff
@@ -0,0 +1,51 @@
+# Bug Fix 
+**Name:** 
+
+**Issue Reference:**
+
+
+## Modified
+
+**Class/Function Name:** *ClassName* 
+
+**module:** *python_filename.py*
+
+**package:** */directory_name*
+
+*Please provide any other information on what is modified to fix this bug*
```

Comment:
Is all of this necessary, since this information is clearly available under 'files changed'?

Reply:
It would be good to know at least directory as I sometimes skim the file name/ directory at the review part like how Borealis made it's own directory and that goes against the flat directory system I was aiming to do. However that another issue to discuss later. 

Also it addresses what is the major change in this PR.